core/trino-main/src/main/java/io/trino/connector/StaticCatalogManager.java

```diff
@@ -200,6 +200,13 @@ public Optional<CatalogProperties> getCatalogProperties(CatalogHandle catalogHan
         return Optional.empty();
     }
 
+    @Override
+    public Set<CatalogHandle> getActiveCatalogs()
+    {
+        //static catalog manager does not differentiate between active and not for management between machines
```

Comment:
put a space after he double slash `//`

Also, I don't understand what "not for management between machines" means